src/bp/core/services/cms.ts

```diff
@@ -446,6 +446,13 @@ export class CMSService implements IDisposeOnExit {
         ...args,
         ...content.formData
       }
+    } else {
```

Comment:
there's a good old keyword for that `else if`

Reply:
haha, didn't even realize i wrote that

Reply:
haha, didn't even realize i wrote that :P rookie mistake